REFERENCE.md

```diff
@@ -766,15 +766,15 @@ Default value: 65536
 
 ##### `unixsocket`
 
-Data type: `Stdlib::Absolutepath`
+Data type: `Variant[Stdlib::Absolutepath, Enum['']]`
```

Comment:
With this data type the PR is backward-incompatible because it is not possible to use `false` to disable unix socket.

Reply:
I marked the original PR as backwards-incompatible. This particular PR is backwards compatible.